### ISSUE TITLE
Adding a screenshot to the README

### ISSUE THREAD

**Issue by alexfertel** [Veykril/MuPanda#1]
I know that this is me being lazy, but would you add a screenshot to the README? I know that you explain the colors in the README and that we have screenshots in the panda repo, but I kinda wanna know *what* is different in your theme.

I have used the panda theme before and found it was quite nice. So When I saw this I instantly got curious :grin:

**Comment by Veykril** [comment]
I can try making a better screenshot later and add that.
Sample screenshot for now:
![Code_Z6zaAnq8PT](https://user-images.githubusercontent.com/3757771/126920084-f5e9ff22-1148-4c66-a718-8b83c54cf1d1.png)

The main difference is that this one is tailored towards rust with rust-analyzer semantic highlighting(in fact I didn't even bother setting the non-semantic token colors so opening a project will show some weird coloring until rust-analyzer highlights it).

I also did some other changes here and there that I don't remember as well as switching some colors around I think but the main color palette is the same.

I did add some other languages as can be seen in the repo source though those arent really fleshed out, only quick jobs to get some similar highlighting. Reason for that is mainly that I don't really use languages other than rust.
